docs/release-notes.rst

```diff
@@ -6,6 +6,195 @@
  Release Notes
 ###############
 
+**************
+ Version 0.35
+**************
+
+Version 0.35.0
+==============
+
+**Release Date:** August 06, 2024
```

Comment:
August 9th?

Reply:
probably a safe bet, worst case scenario; I'll bump it back to the 8th tomorrow if things wrap up 😄